downstream/assemblies/eda/assembly-eda-user-guide-overview.adoc

```diff
@@ -24,7 +24,7 @@ The following procedures form the user configuration:
 [NOTE]
 
 ====
-* API documentation for {EDAcontroller} is available at \https://<eda-server-host>/api/eda/v1/docs
+* API documentation for {EDAcontroller} is available at \https://<platform-host>/api/eda/v1/docs
```

Comment:
Im not sure if instead of calling this `platform-host`, if we should instead call this `gateway-host`? 
Perhaps gateway host may be better, since this is more inline with terminology here - https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/rpm_installation/assembly-platform-install-scenario#con-install-scenario-recommendations

Reply:
LGTM